PowerPointLabs/PowerPointLabs/ShapesLab/Views/ShapesLabCategoryInfoDialogBox.xaml.cs

```diff
@@ -1,4 +1,5 @@
-﻿using System.Text.RegularExpressions;
+﻿using System.Collections.Generic;
```

Comment:
Thanks for the PR! Looks good to me but was wondering if this `using System.Collections.Generic;` is really needed or not?